traitsui/testing/tester/ui_wrapper.py

```diff
@@ -73,6 +73,44 @@ def __init__(self, target, registries, delay=0):
         self._registries = registries
         self.delay = delay
 
+    def help(self):
+        """ Print help messages.
+        (This function is intended for interactive use.)
+        """
+        interactions = dict()
+        locations = dict()
+        for registry in self._registries[::-1]:
```

Comment:
im not sure why we are iterating in reverse order

Reply:
Yep, this needs a test.

Reply:
IMO it is still worth adding a one line comment here that reversing means ordering based on priority.